src/components/organisms/OcModal/OcModal.vue

```diff
@@ -459,6 +488,16 @@ export default {
       />
     </template>
   </oc-modal>
+  <oc-modal
+    icon="information"
+    title="Accept terms of use"
+    message="Do you accept our terms of use?"
+    button-cancel-text="Decline"
+    button-confirm-text="Accept"
+    checkbox="true"
+    checkbox-label="I accept the terms of use"
```

Comment:
haha, actually very comprehensible, yet annoying use case. Good choice for the docs 👍